lib/ansible/module_utils/yumdnf.py

```diff
@@ -44,9 +44,7 @@
         update_cache=dict(type='bool', default=False, aliases=['expire-cache']),
         update_only=dict(required=False, default="no", type='bool'),
         validate_certs=dict(type='bool', default=True),
-        # this should not be needed, but exists as a failsafe
-        lock_poll=dict(type='int', default=-1),
```

Comment:
I don't think we can just remove this now that it has been in a release.  We'll have to deprecate it, potentially by still accept it but warn when it is given.

Reply:
Has it been in a release? It said added in 2.8, so I assumed it hadn't.

Reply:
My mistake.  Timelines are beginning to blur for me.